OpenMLDB/src/src/main/java/com/_4paradigm/benchmark/Util.java

```diff
@@ -31,49 +33,65 @@
 import com._4paradigm.openmldb.proto.Type;
 import com._4paradigm.openmldb.sdk.SqlExecutor;
 
+import org.slf4j.*;
+import org.apache.log4j.PropertyConfigurator;
+
 public class Util {
 
+
     public static boolean executeSQL(String sql, SqlExecutor executor) {
+        System.out.println(sql);
         java.sql.Statement state = executor.getStatement();
         try {
             boolean ret = state.execute(sql);
         } catch (Exception e) {
-            e.printStackTrace();
-            return false;
+            throw new RuntimeException("", e);
         }
         return true;
     }
     // Only for load
     public static boolean executeSQL_block(String sql, SqlExecutor executor) {
+        System.out.println(sql);
         java.sql.Statement state = executor.getStatement();
         try {
             boolean ret = state.execute(sql);
             // check status
             ResultSet res = state.getResultSet();
             res.next();
+            // print the job info
             System.out.println("[INFO LOG]:\n" + res.getString(1));
             String jobID = extractJobID(res.getString(1));
-            System.out.println("[INFO LOG]:" + jobID);
-            boolean retOfShow = state.execute("SHOW JOB " + jobID);
+            boolean retOfShow = state.execute("SHOW JOB " + jobID + ";");
             ResultSet resOfShow = state.getResultSet();
             resOfShow.next();
+            
+            // Format the date and time using a formatter
+            DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd HH:mm:ss");
+            
             while(!resOfShow.getString(3).toLowerCase().equals("finished") &&
                 !resOfShow.getString(3).toLowerCase().equals("lost") &&
                 !resOfShow.getString(3).toLowerCase().equals("killed") &&
                 !resOfShow.getString(3).toLowerCase().equals("failed")) {
-                System.out.println("[INFO LOG]:" + resOfShow.getString(6) + "///" + resOfShow.getString(3));
+                // heartbeat
+                System.out.println("[HEART BEAT]: " + LocalDateTime.now().format(formatter) + " Job" + jobID + " " + resOfShow.getString(3));
```

Comment:
change info log to heart beat, it doesn't help to reduce the log, just a name change. Why not use log? debug log, and if user don't need it, set level to info. And you can use `org.slf4j.Marker` to impl log every N.